includes/CLI/Plugin_Check_Command.php

```diff
@@ -355,12 +355,13 @@ public function list_check_categories( $args, $assoc_args ) {
 	private function get_check_categories() {
 		$check_categories = new Check_Categories();
 		$categories_slugs = $check_categories->get_categories();
+		$category_labels  = $check_categories->get_category_labels();
 
 		return array_map(
-			function ( $slug ) {
+			function ( $slug ) use ( $category_labels ) {
 				return array(
 					'slug' => $slug,
-					'name' => ucfirst( str_replace( '_', ' ', $slug ) ),
+					'name' => $category_labels[ $slug ],
 				);
 			},
 			$categories_slugs
```

Comment:
Nit-pick: We could use only `$category_labels` here as well as it contains both the slugs and the labels.